components/Rate/src/index.svelte

```diff
@@ -62,7 +62,7 @@
 			text = (isString(texts) ? texts : textsMap[Math.ceil(currentValue)]) as string;
 		} else if (showScore) {
 			text = scoreTemplate.replace(
-				/\{\s*value\s*\}/,
+				/\{\s*value\s*\}/g,
```

Comment:
Will this fix the warning for the icon?

Reply:
> Will this fix the warning for the icon?

No, I can't fix it yet

Reply:
> Will this fix the warning for the icon?

This is done to handle the situation where {value} appears multiple times.

Reply:
For the icon, try another way to put it in the shortcut so that it will eventually be placed in the safelist instead of using the unocss icon class directly. I suspect that the build tool ignores these icons.